### PR TITLE
Feature/gas price

### PR DESCRIPTION
**Pull request by cshcomcom**

### CODE REVIEW

File: worker_node/job/send.js [hunk 1]
```diff
@@ -149,12 +149,13 @@ class Send extends Base {
...
   }
 
-  makeBaseTransaction() {
+  makeBaseTransaction(gasPrice) {
     return {
       operation: {
         ...this.config.transactionOperation,
       },
       nonce: -1,
+      gas_price: gasPrice,
```
Comment (liayoo) [inline worker_node/job/send.js:158]: minor) how about just using this.config.gasPrice || 0 here without passing it as an arg?

Comment (cshcomcom) [reply worker_node/job/send.js]: Looks better! Done

**Review by platfowner — APPROVED** [review]
LGTM
Thanks!

**Review by liayoo — APPROVED** [review]
LGTM, thanks!

**Comment by cshcomcom** [comment]
Thanks for reviews! Now merging

**Pull request merged**